app/views/customers/show.html.erb

```diff
@@ -0,0 +1,15 @@
+<h2><%= @customer.name %></h2>
+<br>
+<% @items.each do |item| %>
+  <section id="item-<%= item.id %>">
+    <p><%= item.name %>, $<%= item.price %>, <%= item.supermarket_name %></p>
+  </section>
+<% end %>
+<br>
+<h4>Add item</h4>
+<br>
+<%= form_with url:"/customers/#{@customer.id}", method: :post, data: {turbo: false} do |form| %>
```

Comment:
To make this route more RESTful we would want to add `items` to the end of this:
```ruby
"/customers/#{customer.id}/items"
```
This creates the joining of an item with a customer.